apps/frontend/src/app/5_pages/AavePage/components/LendPositionsList/components/WithdrawModal/WithdrawForm.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  AmountInput,
+  Button,
+  ErrorBadge,
+  ErrorLevel,
+  Select,
+  SimpleTable,
+  SimpleTableRow,
+  Tabs,
+  TabType,
+} from '@sovryn/ui';
+import { Decimal } from '@sovryn/utils';
+
+import { AmountRenderer } from '../../../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { AssetRenderer } from '../../../../../../2_molecules/AssetRenderer/AssetRenderer';
+import { useDecimalAmountInput } from '../../../../../../../hooks/useDecimalAmountInput';
+import { translations } from '../../../../../../../locales/i18n';
```

Comment:
doesn't the project have absolute imports?

Reply:
Unfortunatelly no, pretty crazy urls really but is like this all over the place. Example -> https://github.com/DistributedCollective/sovryn-dapp/blob/develop/apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameBalance/LendFrameBalance.tsx